### ISSUE TITLE
Trying to log in causes a "404 not found" error

### ISSUE THREAD

**Issue by migitri** [benjamindean/observatory#15]
I'm trying to log into Observatory for the first time, but when I click the button on the page the app takes me to on Steam or ITAD to authorize my account, it only leads to a "404 not found" error. I tried going straight to `https://getobservatory.app/app/auth/itad` as well, but that also threw a "404 not found" error.

I'm using it a Pixel 6 Pro with Android 15, if that's relevant. I'm using the most recent version of Observatory.

**Comment by benjamindean** [comment]
Looks like app links are broken out of the box. Try going to the app settings (in Android) and there should be "Open by default" category or similar. Inside it, look for the checkbox with the app's domain and enable it.

I will look into the issue. Thanks for reporting it!

**Comment by migitri** [comment]
@benjamindean Thank you! That setting was already turned on, but I tried switching my default browser from Firefox to Edge (just picked one at random that I still had installed) as well, just to see if it would work. Once I signed in, it asked me if I wanted to go to Observatory. Then it worked. It turns out that Firefox was set to never open links in other apps. I changed that setting and the login worked in Firefox too after that.

So it turns out that it was user error rather than a bug. Sorry for the mistake!
